packages/voilite/src/plugins.ts

```diff
@@ -0,0 +1,72 @@
+/***************************************************************************
+ * Copyright (c) 2022, Voilà contributors                                   *
+ * Copyright (c) 2022, QuantStack                                           *
+ *                                                                          *
+ * Distributed under the terms of the BSD 3-Clause License.                 *
+ *                                                                          *
+ * The full license is in the file LICENSE, distributed with this software. *
+ ****************************************************************************/
+import * as base from '@jupyter-widgets/base';
+import {
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { IThemeManager } from '@jupyterlab/apputils';
+import { PromiseDelegate } from '@lumino/coreutils';
+import { translatorPlugin, pathsPlugin } from '@voila-dashboards/voila';
```

Comment:
The common pattern in JupyterLab for now has been to separate plugins from the core package (in their `-extension` package).

I don't have a strong opinion about this for Voila for now, but maybe something to consider when iterating on `voilite`.